SETUP/lint_charsuites.php

```diff
@@ -19,6 +20,24 @@
         exit(1);
     }
 
+    // Validate that the character suite does not contain codepoints we convert
+    // to ASCII
+    $disallowed_characters = convert_codepoint_ranges_to_characters(get_disallowed_codepoints());
+    $charsuite_characters = convert_codepoint_ranges_to_characters($charsuite->codepoints);
+    $characters = array_intersect($disallowed_characters, $charsuite_characters);
+    if($characters != [])
+    {
+        echo "ERROR: disallowed characters found in suite\n";
+        foreach($characters as $char)
+        {
+            echo sprintf("    %s: %s\n", $char, utf8_chr_to_hex($char));
+        }
+        exit(1);
+    }
+
+    // Confirm that all hex codes are in lowercase
+    validate_hex_codes_lowercase($charsuite->codepoints, "uppercase hex values found in charsuite codepoints");
```

Comment:
And speaking of translation, does this string need to be wrapped in `_()`?

Reply:
We don't translate files in `SETUP/` since they will never be seen by the user -- just site admins when they set up the site or developers.